package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@exodus/react-native-passkeys",
-  "version": "0.3.8",
+  "name": "@passkeys/react-native-passkeys",
```

Comment:
discussed with @andrejborstnik to move this to the passkeys scope to avoid version conflicts